src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/instrument-your-own.mdx

```diff
@@ -145,6 +145,8 @@ custom:
     title="Terraform"
   >
     Terraform is a popular general-purpose infrastructure as code tool. It can be used to manage AWS resources. We offer [some examples](https://github.com/newrelic/newrelic-lambda-extension/tree/main/examples/terraform) of New Relic instrumented Lambda functions deployed using Terraform scripts.
+
+    You can use the [New Relic Terraform Provider](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/cloud_aws_integrations) to set up your AWS integrations.
```

Comment:
Should it be "New Relic Terraform provider"?

Reply:
I saw this capitalized in a comment on the Terraform registry, but it does look like a provider is just a generic term. So, I'll make this lower case.